core/include/io.h

```diff
@@ -40,6 +40,17 @@ static inline void io_write64(vaddr_t addr, uint64_t val)
 	*(volatile uint64_t *)addr = val;
 }
 
+static inline void write_64bit_pair(uintptr_t addr, uint64_t val_l,
```

Comment:
This function should go into `core/arch/arm/include/arm64.h`, with the `dmb oshst` instruction excluded.
Something along:
``` diff
--- a/core/arch/arm/include/arm64.h
+++ b/core/arch/arm/include/arm64.h
@@ -355,6 +355,12 @@ static inline __noprof void tlbi_vale1is(uint64_t va)
 	asm volatile ("tlbi	vale1is, %0" : : "r" (va));
 }
 
+static inline void write_64bit_pair(uint64_t dst, uint64_t hi, uint64_t lo)
+{
+	asm volatile ("stp	%0, %1, [%2]" : :
+		      "r" (hi), "r" (lo), "r" (dst) : "memory" );
+}
+
```
Note that the high argument should be before the low to harmonize with for instance `reg_pair_to_64()`.

Reply:
Done, Thank you adivce.